integrations/docker/images/chip-build-android/run.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
this should be a symbolic link up 2 directories?

Reply:
thanks, done